src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -0,0 +1,435 @@
+﻿namespace Equinox.MessageDb
+
+open Equinox.Core
+open Equinox.MessageDb.Core
+open FSharp.Control
```

Comment:
alphabetic order violation (if we're being case insensitive anyway!)

Reply:
I did a vim `:sort` to get this order 😓. I'll move it around

Reply:
ha, I do them all manually; and I always ponder it as I put FS beside Fs !